cypresse2e/cypress/integration/e2eTests/offerHelp/offerHelpQuestion.spec.js

```diff
@@ -0,0 +1,109 @@
+            var nextButton = offerHelpQuestion.getNextButtonOne();
+            nextButton.should('be.visible').click();
+        });
+
+    });
+
+    context('User goes to offer help question 2', () => {
+        beforeEach(() => {
+            offerHelpQuestion.visitOfferHelpTwo();
+        });
+
+        it('Offer Help Question Two Page contains heading & question two', () => {
+            var questionNumberTwo = "Question 2/3";
+            var questionTwo = "Where are you located?";
+            var infoText = "We want to show you the most relevant results";
+            offerHelpQuestion.getQuestionNumberTwo().should('be.visible').contains(questionNumberTwo);
+            offerHelpQuestion.getQuestionTwo().should('be.visible').contains(questionTwo);
+            offerHelpQuestion.getInfoText().should('be.visible').contains(infoText);
+
+        });
+
+        it('Offer Help Question Two Page - location can be typed and chosen from a list of options from the typeahead dynamic dropdown', () => {
+            var locationInputLine = offerHelpQuestion.getLocationInputLine();
+            locationInputLine.should('be.visible').click();
+            locationInputLine.type("Los")
+            offerHelpQuestion.getLocationDropDown().should('be.visible');
+            offerHelpQuestion.getLocationDropDown().find('div.ant-select-item-option').contains('Los Gatos, CA, USA').click();
+        });
+
+        it('Offer Help Question Two Page - location input line Information is visible', () => {
+            var locationInputInfo = "Enter address, zip code, or city";
+            offerHelpQuestion.getLocationInputInfo().should('be.visible');
+            offerHelpQuestion.getLocationInputInfo().contains(locationInputInfo);
+        });
+
+        it('Offer Help Question Two Page - Share My Location Icon and Text is visible and clickable', () => {
+            var shareMyLocationText = "Share My Location";
+            var shareMyLocationIcon = offerHelpQuestion.getShareMyLocationIcon();
+            shareMyLocationIcon.should('be.visible').click({ force: true });
+            offerHelpQuestion.getShareMyLocationText().should('be.visible').contains(shareMyLocationText).click({ force: true });
+
+        });
+
+        it('Offer Help Question Two Page - Show Postings Link is visible and clickable', () => {
+            var showMePostLink = "Show me postings from anywhere";
+            offerHelpQuestion.getShowMePostLink().should('be.visible').contains(showMePostLink).click({ force: true });
+        });
+
+        it('Offer Help Question Two Back Button is visible and clickable', () => {
+            var backButton = offerHelpQuestion.getBackButtonTwo();
+            backButton.should('be.visible').click();
+
+        });
+
+        it('Offer Help Question One Next Button is visible and clickable', () => {
+            var nextButton = offerHelpQuestion.getNextButtonTwo();
+            nextButton.should('be.visible').click();
+        });
+
```

Comment:
The offer help next button for Question 2 has just been removed and merged to staging. Please refer to PR [https://github.com/FightPandemics/FightPandemics/pull/1525](). You'll need to pull the most recent updates and merge to your branch, remove this snippet of code since it's not necessary anymore, then push out your branch again with the updates.

Reply:
Will push again with the new updates.